AlphaWallet/Protection/Coordinators/ProtectionCoordinator.swift

```diff
@@ -21,7 +21,7 @@ class ProtectionCoordinator: Coordinator {
 	}
 
 	func didFinishLaunchingWithOptions() {
-		splashCoordinator.start()
+		//splashCoordinator.start()
```

Comment:
Commented out code should generally not be checked in, they should be deleted.

What is the rationale for removing this though?

Reply:
Actually this part to code is adding splash view on window even before app is loaded so its not getting proper frame coordinates. because of that when we open app after creating wallet it shows black screen with logo with layout issue.
`splashCoordinator.start()`
As its already called on `applicationDidBecomeActive` we can remove this line of code in 
`didFinishLaunchingWithOptions`.